### PR TITLE
rework USB UX

### PR DESCRIPTION
**Pull request by benadida**
This brings over the `USBControllerButton` approach from election-manager, with the actual work done in `AppRoot`, and the use of a context to provide status to the button.

### CODE REVIEW

File: src/components/#USBControllerButton.tsx# [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react'
```
Comment (beausmith) [inline src/components/#USBControllerButton.tsx#:1]: Why does this file name have `#` prefix and suffix?

**Review by beausmith — APPROVED** [review]
One question and 🙃 on reduced test coverage percentage.

File: src/components/#USBControllerButton.tsx# (hunk 1 above)
Comment (benadida) [reply src/components/#USBControllerButton.tsx#:1]: ummm, stupid backup file, I need to delete it.

**Comment by benadida** [comment]
> One question and  on reduced test coverage percentage.

On the test coverage: I don't have an easy way to fix this in percentage, although this is relatively well tested.

**Pull request merged**